pkg/signatory/signatory.go

```diff
@@ -52,6 +52,7 @@ type PublicKey struct {
 	VaultName     string
 	ID            string
 	Policy        *config.TezosPolicy
+    Status        string
```

Comment:
Indentation is off here. You must always run `gofmt`, please configure it in your editor to run automatically. :)

Reply:
fixed. The gofmt is very helpful!